service/service.go

```diff
@@ -0,0 +1,113 @@
+package service
+
+import (
+	"fmt"
+	"github.com/companieshouse/innovation-accounts-statistics-tool/config"
+	"github.com/companieshouse/innovation-accounts-statistics-tool/models"
+	"os"
+	"time"
+
+	"github.com/companieshouse/innovation-accounts-statistics-tool/db"
+	log "github.com/sirupsen/logrus"
+)
+
+const statisticsReportFileNamePrefix = "CHS_SmallFullAccounts_Statistics"
+const csvFileSuffix = ".csv"
+
+// Service provides an interface to retrieve statistics in a CSV format.
+type Service interface {
```

Comment:
Can we add another function to this interface called `Shutdown()` please?

The implementation of this method should just call `service.db.Shutdown()` which is exposed [here](https://github.com/companieshouse/innovation-accounts-statistics-tool/pull/1/files#diff-620a5a7dfa1eb7c470357fcb4ccc00492bb882bd5d4e432b100370a05cc9369cR104)

Then in `lambda.go`, at the end of the `Execute` function, call `lambda.service.Shutdown()`

This will terminate connection to the database gracefully